code/World/Shared/WorldRendererShared.cpp

```diff
@@ -520,7 +520,17 @@ void WorldRendererShared::setupLightPass(
 							0.0f,
 							1.0f
 						);
-						renderContext->draw(svrb);	
+						renderContext->draw(svrb);
+
+						// Set scissor.
+						auto ssrb = renderContext->alloc< render::SetScissorRenderBlock >();
+						ssrb->scissor = render::Rectangle(
+							svrb->viewport.left,
+							svrb->viewport.top,
+							svrb->viewport.width,
+							svrb->viewport.height
+						);
+						renderContext->draw(ssrb);
```

Comment:
Why is this necessary?

Reply:
I'm not sure if this is necessary tbh. 
Since the vp is being changed here, the scissor and vp will have different sizes.
I assume setting the scissor to the same size as the viewport will preserve existing behaviour.

Reply:
I opted to revert this change.